src/server/node_func.c

```diff
@@ -524,8 +524,8 @@ initialize_pbsnode(struct pbsnode *pnode, char *pname, int ntype)
 	pnode->nd_attr[(int)ND_ATR_Sharing].at_flags =
 		ATR_VFLAG_SET|ATR_VFLAG_DEFLT;
 
-	pnode->nd_attr[(int)ND_ATR_vnode_pool].at_val.at_long = 0;
-	pnode->nd_attr[(int)ND_ATR_vnode_pool].at_flags =
+//	pnode->nd_attr[(int)ND_ATR_vnode_pool].at_val.at_long = 0;
+//	pnode->nd_attr[(int)ND_ATR_vnode_pool].at_flags =
 		ATR_VFLAG_SET|ATR_VFLAG_DEFLT;
```

Comment:
Shouldn't this line also be commented ?

Reply:
Code shouldn't be commented out.  It should either be uncommented or removed.